response-to-audit-report.md

```diff
@@ -5,20 +5,20 @@
+Marlowe is a domain-specific language (DSL) for multi-party financial contracts. Operational semantics are formalized in the Isabelle proof-assistant and an interpreter for Marlowe for the Cardano blockchain is provided as Plutus validator script. Early in 2023 the Marlowe specification, the on-chain implementation and the validator test-suite were audited by Tweag's High Assurance Software Group. The report is available online. It categorizes findings of either high, medium or low severity. The present document summarizes the subsequent changes made to the Marlowe specification, the implementation of the validator and the property-based tests for the validator. All of the high-severity findings have been remedied or mitigated through changes to the specification, the proofs in Isabelle or the implementation. Nearly all of the medium-severity and low-severity findings have also been addressed, otherwise justification for not addressing a finding has been provided.
 
-The high-severity findings related to handling of negative deposits, prevention of "double satisfaction", enforcement of state invariants, an implementation difference between the formal specification versus the Plutus implementation, and the Isabelle proof of money preservation. Marlowe's Plutus validator has been altered to resolve these first three findings so that it correctly prevents negative deposits from altering the balance of internal accounts, it only allows other Plutus scripts to run during transactions that do not make Marlowe payments, and it rigorously enforces invariants for initial and final states. The fourth finding regarding the Plutus implementation was mitigated via code analysis and property-based testing. The fifth finding about the money-preservation proof was remedied by revising the Isabelle proof.
+The high-severity findings comprise handling of negative deposits, prevention of "double satisfaction", enforcement of state invariants, an implementation difference between the formal specification versus the Plutus implementation and the proof of money preservation in Isabelle. The Marlowe validator has been altered to resolve these first three findings so that it correctly prevents negative deposits from altering the balance of internal accounts, it only allows other Plutus scripts to run during transactions that do not make Marlowe payments and it rigorously enforces invariants for initial and final states. The fourth finding regarding the Plutus implementation was mitigated via code analysis and property-based testing. The fifth finding about the proof of the money preservation theorem was remedied by revising the Isabelle code.
```

Comment:
The line could probably be removed, as the `Overview of ... changes` contains the same information again.